README.adoc

```diff
@@ -0,0 +1,20 @@
+JSON for creating connection http://localhost:8083/connectors
+
+`{
+"name": "pg-connector",
+"config": {
+"connector.class": "io.debezium.connector.postgresql.PostgresConnector",
+"plugin.name": "pgoutput",
+"database.hostname": "host.docker.internal",
+"database.port": "5432",
+"database.user": "root",
+"database.password": "root",
+"database.server.id": "100",
+"database.server.name": "orders_server",
+"database.dbname" : "order_service",
+"table.include.list":"public.outbox",
+"database.history.kafka.bootstrap.servers": "kafka:9092",
+"transforms": "unwrap",
+"transforms.unwrap.type": "io.debezium.transforms.ExtractNewRecordState"
+}
+}`
```

Comment:
1) create the folder `doc`
2) Using Postman,  configure request.
3) export postman collection as a single file
4) set this file to `doc` folder

Reply:
done